### PR TITLE
Develop into master

### PR DESCRIPTION
**Pull request by fabian-steinberg**

### CODE REVIEW

File: dist/css/bootstrap-multiselect.min.css [hunk 1]
```diff
@@ -1,44 +1 @@
...
...er,.multiselect-container .multiselect-group:focus,.multiselect-container .multiselect-group:hover,.multiselect-container .multiselect-option:focus,.multiselect-container .multiselect-option:hover{background-color:#a9a9a9!important}.multiselect-container .multiselect-all .form-check,.multiselect-container .multiselect-group .form-check,.multiselect-container .multiselect-option .form-check{padding:0 5px 0 20px}.multiselect-container .multiselect-all:focus,.multiselect-container .multiselect-group:focus,.multiselect-container .multiselect-option:focus{outline:0}.form-inline .multiselect-container span.form-check{padding:3px 20px 3px 40px}.input-group.input-group-sm>.multiselect-native-select .multiselect{padding:.25rem .5rem;font-size:.875rem;line-height:1.5;padding-right:1.75rem;height:calc(1.5em + .5rem + 2px)}.input-group>.multiselect-native-select{flex:1 1 auto;width:1%}.input-group>.multiselect-native-select>div.btn-group{width:100%}.input-group>.multiselect-native-select:not(:first-child) .multiselect{border-top-left-radius:0;border-bottom-left-radius:0}.input-group>.multiselect-native-select:not(:last-child) .multiselect{border-top-right-radius:0;border-bottom-right-radius:0}
```
Comment (s-eckard) [inline dist/css/bootstrap-multiselect.min.css:1]: The comment block above is missing

File: dist/js/bootstrap-multiselect.js [hunk 2]
```diff
@@ -463,7 +463,7 @@
                 buttonGroupReset: '<button type="button" class="multiselect-reset btn btn-secondary btn-block"></button>',
                 option: '<button type="button" class="multiselect-option dropdown-item"></button>',
                 divider: '<div class="dropdown-divider"></div>',
-                optionGroup: '<button type="button" class="multiselect-group dropdown-item"></button>',
+                optionGroup: '<button type="button" class="multiselect-group dropdown-item pl-1"></button>',
```
Comment (s-eckard) [inline dist/js/bootstrap-multiselect.js:466]: Was this change added to the documentation of the templates?

File: dist/js/bootstrap-multiselect.min.js [hunk 3]
```diff
@@ -1,44 +1 @@
...
...boxes},updateOriginalOptions:function(){this.originalOptions=this.$select.clone()[0].options},asyncFunction:function(callback,timeout,self){var args=Array.prototype.slice.call(arguments,3);return setTimeout((function(){callback.apply(self||window,args)}),timeout)},setAllSelectedText:function(allSelectedText){this.options.allSelectedText=allSelectedText,this.updateButtonText()},isFirefox:function(){var firefoxIdentifier="firefox",valueNotFoundIndex=-1;return!(!navigator||!navigator.userAgent)&&navigator.userAgent.toLocaleLowerCase().indexOf("firefox")>-1},createAndApplyUniqueId:function($relatedElement){var id="multiselect_"+this.multiselectId+"_"+this.internalIdCount++;return $relatedElement&&($relatedElement[0].dataset.multiselectid=id),id},generateUniqueId:function(){return Math.random().toString(36).substr(2)}},$.fn.multiselect=function(option,parameter,extraOptions){return this.each((function(){var data=$(this).data("multiselect"),options;data||(data=new Multiselect(this,"object"==typeof option&&option)),"string"==typeof option&&data[option](parameter,extraOptions)}))},$.fn.multiselect.Constructor=Multiselect,$((function(){$("select[data-role=multiselect]").multiselect()}))}));
```
Comment (s-eckard) [inline dist/js/bootstrap-multiselect.min.js:1]: Also here the comment block is missing

File: dist/css/bootstrap-multiselect.min.css (hunk 1 above)
Comment (fabian-steinberg) [reply dist/css/bootstrap-multiselect.min.css:44]: Comment has been added back

File: dist/js/bootstrap-multiselect.js (hunk 2 above)
Comment (fabian-steinberg) [reply dist/js/bootstrap-multiselect.js:466]: Changes have now been added to documentation

File: dist/js/bootstrap-multiselect.min.js (hunk 3 above)
Comment (fabian-steinberg) [reply dist/js/bootstrap-multiselect.min.js:44]: Also has been put back

**Pull request merged**